runtime/vm/UpcallExceptionHandler.cpp

```diff
@@ -55,7 +55,7 @@ ffiCallWithSetJmpForUpcall(J9VMThread *currentThread, ffi_cif *cif, void *functi
 ffiCallWithSetJmpForUpcall(J9VMThread *currentThread, ffi_cif *cif, void *function, UDATA *returnStorage, void **values)
 #endif /* FFI_NATIVE_RAW_API */
 {
-	jmp_buf jmpBufferEnv = {0};
+	jmp_buf jmpBufferEnv = {};
```

Comment:
Reading the documentation i see that `{ }` was introduced in C23:  https://en.cppreference.com/w/c/language/array_initialization

Reply:
I cannot find when initializing an array with `{}` in c++ started to be valid.

Reply:
Perhaps we should omit `=` to use a form that has been valid since C++11?
```c
	jmp_buf jmpBufferEnv {};
```

Reply:
Whatever you decide on, pls try it in a vmfarm build on AIX and z/OS before merging.

Reply:
I was going to suggest that until I noticed that this code is guarded by `#if JAVA_SPEC_VERSION >= 16`.

Reply:
I think the syntax I have is okay even before C++-11, judging from the articles [initialization](https://en.cppreference.com/w/cpp/language/initialization) and [aggregate_initialization](https://en.cppreference.com/w/cpp/language/aggregate_initialization) - it's either aggregate initialization or list initialization of an aggregate type (an array type). I don't think the C standard applies here.

The page [zero_initialization](https://en.cppreference.com/w/cpp/language/zero_initialization) does point out that `jmp_buf jmpBufferEnv {};` will work, as you say.

Reply:
I thought you agree with my suggestion (https://github.com/eclipse-openj9/openj9/pull/19804#discussion_r1664767554), but this was merged without that change being made. What happened?